pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -3770,6 +3770,21 @@ public boolean isDispatcherPauseOnAckStatePersistentEnabled() {
         return b == null ? false : b.booleanValue();
     }
 
+    @Override
+    public void updateDispatchPauseOnAckStatePersistentEnabled() {
+        super.updateDispatchPauseOnAckStatePersistentEnabled();
+        // Trigger new read if subscriptions has been paused before.
+        if (!brokerService.getPulsar().getConfiguration().isDispatcherPauseOnAckStatePersistentEnabled()) {
+            updateDispatchPauseOnAckStatePersistentEnabled();
```

Comment:
would this cause recursion? do you need `super.` prefix?

Reply:
Ah, It duplicates with `line-3775` and has been deleted